six/modules/c++/six.sicd/source/GeoData.cpp

```diff
@@ -114,20 +114,30 @@ bool GeoData::validate(logging::Logger& log) const
     bool valid = true;
     std::ostringstream messageBuilder;
 
-    // 2.10
-    scene::LLAToECEFTransform transformer;
-    Vector3 derivedEcf = transformer.transform(scp.llh);
-    double ecfDiff = (scp.ecf - derivedEcf).norm();
-
-    if (ecfDiff > ECF_THRESHOLD)
+    if (Init::isUndefined(scp.llh) ||
```

Comment:
Yes. I ran into a couple segfaults after I made the request and thought it would fit.